source/manual/covid-19-services.html.md

```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
 section: Services
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-06-03
+last_reviewed_on: 2020-10-15
 review_in: 1 month
 ---
 
-GOV.UK operates three standalone services for COVID-19 response:
+Between March and October 2020 the GOV.UK operated three standalone
+services for the COVID-19 pandemic response:
 
-- Offer coronavirus support from your business
-- Get coronavirus support as an extremely vulnerable person
-- Find support if you're affected by coronavirus
+- [Business volunteer service](#business-volunteer-service)
+- [Vulnerable people service](#vulnerable-people-service)
+- [Find support service](#find-support-service)
 
-## Offer coronavirus support from your business
+These services are no longer deployed, live or in active development.
 
-This service allows businesses to tell us how they might be able to
-help with the response to coronavirus.  This may include goods or
+## Business volunteer service
+
+This service allowed businesses to tell us how they might be able to
+help with the response to coronavirus.  That included goods or
 services such as medical equipment, hotel rooms or childcare.
 
-- [Start page](https://www.gov.uk/coronavirus-support-from-business)
-- [Service](https://coronavirus-business-volunteers.service.gov.uk/medical-equipment)
-- [GitHub](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form)
+**The service has been replaced with a guidance page.**
+
+A backup of the production database was taken prior to the service
+being removed. It is available in the `govuk-production-database-backups`
+AWS S3 bucket named `coronavirus-business-volunteer-form/production.sql.gzip`
+with 365 day retention period (expiry date: Wed, 13 Oct 2021 00:00:00
+GMT) as per our privacy policy.
```

Comment:
This paragraph isn't very easy to read on the page as it contains long names and a date that cross line breaks. I'd suggest breaking it down into bullets. Something like:
```
A backup of the production database was taken prior to the service
being removed. It is available at:

- AWS S3 bucket - `govuk-production-database-backups`
- Key - `coronavirus-business-volunteer-form/production.sql.gzip`
- Retention period - 365 days (as per our privacy policy)
- expiry date - Wed, 13 Oct 2021 00:00:00 GMT
```

Reply:
Thanks @reggieb - fixed 👍